apps/omg/lib/omg/root_chain_coordinator.ex

```diff
@@ -72,10 +72,6 @@ defmodule OMG.RootChainCoordinator do
   end
 
   def init({args, configs_services}) do
-    {:ok, {args, configs_services}, {:continue, :setup}}
-  end
-
-  def handle_continue(:setup, {args, configs_services}) do
```

Comment:
Curious shouldn't we move work from init to `handle_continue/2` instead of the other way around, so the supervisor can move on and start the next child sooner?

Reply:
Mainly the concern in this specific init is `EthereumHeight.get()`

Reply:
Ah reading https://github.com/omgnetwork/elixir-omg/pull/1558#issuecomment-637282889 and re-reading the description, this is how the PR tackles the race condition, my bad 🤦 

I'm thinking through this again